lib/stats/stats-prometheus.c

```diff
@@ -93,6 +94,13 @@ stats_format_prometheus_format_value(const StatsClusterKey *key, StatsCounterIte
       converted_int *= 60;
     case SCU_MINUTES:
       converted_int *= 60;
+    case SCU_SECONDS:
+      if (key->formatting.frame_of_reference == SCFOR_RELATIVE_TO_TIME_OF_QUERY)
+        {
+          GTimeVal now;
```

Comment:
this could be UnixTime still

Reply:
This was unintentional, sorry.

Fixed.